src/config/types.ts

```diff
@@ -96,9 +95,7 @@ namespace CustomTypeConfig {
     strategy: 'versionsByName';
   }
 
-  export type Any = Union.Strict<
-    NameAndVersionProps | NamedVersionString | UnnamedVersionString | VersionsByName
-  >;
+  export type Any = NameAndVersionProps | NamedVersionString | UnnamedVersionString | VersionsByName;
```

Comment:
What was happening when `Union.Strict` was still there please?

Reply:
`Union.Strict` was doing nothing here, because all of these types have the same fields.